ivy/functional/ivy/linear_algebra.py

```diff
@@ -342,14 +345,31 @@ def diagonal(
     ivy.array([1.])
 
     A 3-D Example
```

Comment:
Hi @Sandstorm831 , the edits are looking good so far.
You still need to rename `A 3-D Example` to `Functional Examples`, include the remaining required examples in this section, and also include an `Instance Method Examples` section.

Make sure to refer to the Docstring guide to see exactly what time of examples you need to include:
https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html

Also, have a look at the completed examples for the function add() to get an idea of how the overall formatting should be:
https://github.com/unifyai/ivy/blob/7d510b6cbc71350c6fa22d8273ba5326424eb655/ivy/functional/ivy/elementwise.py#L124

Reply:
> Hi @Sandstorm831 , the edits are looking good so far. You still need to rename `A 3-D Example` to `Functional Examples`, include the remaining required examples in this section, and also include an `Instance Method Examples` section.
> 
> Make sure to refer to the Docstring guide to see exactly what time of examples you need to include: https://lets-unify.ai/ivy/deep_dive/11_docstring_examples.html
> 
> Also, have a look at the completed examples for the function add() to get an idea of how the overall formatting should be:
> 
> https://github.com/unifyai/ivy/blob/7d510b6cbc71350c6fa22d8273ba5326424eb655/ivy/functional/ivy/elementwise.py#L124

Hello,
Thank You for you suggestions, I have added examples of Instances methods, renamed to functional examples and also added more docstring examples.